akka-actor/src/main/scala/akka/actor/ActorSystem.scala

```diff
@@ -82,11 +82,23 @@ object BootstrapSetup {
 
 }
 
-abstract class ProviderSelection private (private[akka] val identifier: String)
+/**
+ * @param identifier the simple name of the selected provider
+ * @param fqcn the fully-qualified class name of the selected provider
+ */
+abstract class ProviderSelection private (
+    private[akka] val identifier: String,
+    private[akka] val fqcn: String,
+    val hasCluster: Boolean)
```

Comment:
Could we make it `private[akka]` as well? Is there a reason to make that public API? (Is there for `ProviderSelection` in the first place?)

Reply:
@jrudolph yes I can not believe I missed that, thanks, pushed.